Gems/RemoteTools/Code/Source/RemoteToolsSystemComponent.cpp

```diff
@@ -315,33 +329,6 @@ namespace RemoteTools
         AZ::SerializeContext* serializeContext;
         AZ::ComponentApplicationBus::BroadcastResult(serializeContext, &AZ::ComponentApplicationBus::Events::GetSerializeContext);
 
-        // Messages targeted at our own application just transfer right over to the inbox.
-        if (target.IsSelf())
```

Comment:
I had to make this change, first to ensure behavior is the same between local and remote (so that there is no bad surprise with message serialization when connecting to remote process) and also because the actual message type is lost if we are not serializing them properly